### ISSUE TITLE
Swipe doesn't work after item removal

### ISSUE THREAD

**Issue by maslianok** [jemise111/react-native-swipe-list-view#109]
![react-native-swipe-list-view-bug](https://cloud.githubusercontent.com/assets/3485490/23891409/d0e709ac-08c8-11e7-86b3-68cf6b984013.gif)

Steps to reproduce:
1. scroll list to the bottom
2. remove any item
3. Swipe doesn't work anymore...

You have to scroll the list to make it work again.

**Comment by hellomaya** [comment]
I have got the same issue, and this issue is happening because of this part code in SwipeRow:

```
line 150
this.props.setScrollEnabled && this.props.setScrollEnabled(false);
```
By default, in SwipeListView it will do this:

```
setScrollEnabled(enable) {
		this._listView.setNativeProps({scrollEnabled: enable});
	}
```

In iOS, when you set it's disable, then touch event stopped too.

https://developer.apple.com/reference/uikit/uiscrollview/1619395-isscrollenabled

I am not fully confirmed the Native props of scrollEnabled is isScrollEnabled in iOS, but by it's explanation:

> If the value of this property is true , scrolling is enabled, and if it is false , scrolling is disabled. The default is true.

> When scrolling is disabled, the scroll view does not accept touch events; it forwards them up the responder chain.

it probably stopped the touch events accidentally, then before we even swipe it to that distant, it stopped.

By comment the line 

```
this.props.setScrollEnabled && this.props.setScrollEnabled(false);
```

The default SwipeListView running just ok.

More explanation and if it will bring some side-effect, still unknown, as I am just figuring it out.

**Comment by dotcink** [comment]
Seems that when you delete an item at the bottom(iOS), the ListView would scroll up a little bit and over its bottom, then unless you restore the unexpected scroll state the Row will never respond touch event.

I resolve it by force `scrollToEnd()` when `yScrollOffset >= contentHeight - layoutHeight` in `onContentSizeChange()`.

```
onLayout={e => this.layoutHeight = e.nativeEvent.layout.height}
onContentSizeChange={(w, h) => this.contentHeight = h}
onScroll={e => this.yScrollOffset = e.nativeEvent.contentOffset.y}
```

**Comment by seclace** [comment]
@dotcink Thank you for solution. But try to change list `dataSource` size programatically so when your `contentHeight - layoutHeight` value becomes `< 0`, and then the scrolling to end is very terrible. So you just need to extend this solution with `yScrollOffset >= contentHeight - layoutHeight && contentHeight - layoutHeight > 0` and only then call `scrollToEnd()`.

**Comment by dotcink** [comment]
@seclace Yeah, you're more accurate.

BTW, what happens for you when `contentHeight - layoutHeight < 0`? For me, `scrollToEnd()` does nothing on that condition.

**Comment by seclace** [comment]
@dotcink the scrollable content is bounces sometimes from top to down, then sometimes from down to top. It's unpredictable :)

**Comment by jemise111** [comment]
Fixed in v1.3.0
